py/torch_tensorrt/dynamo/runtime/_MutableTorchTensorRTModule.py

```diff
@@ -203,6 +211,31 @@ def __init__(
         )
         self.init_finished = True
 
+    def set_dynamic_shape_hint(
+        self,
```

Comment:
Will this be applicable for static shape as well? Would this be a usecase that would be useful?

Reply:
Otherwise we should call it `set_expected_shape_range` or something